samples/openthread/libraries/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+FILE(GLOB app_sources src/*.c)
+
+target_sources(app PRIVATE ${app_sources})
+
+execute_process(
+  COMMAND           git rev-parse HEAD
+  WORKING_DIRECTORY ${openthread_SOURCE_DIR}
+  RESULT_VARIABLE   git_return
+  OUTPUT_VARIABLE   openthread_git_hash)
+
+list(APPEND OT_SETTINGS "OpenThread_commit=${openthread_git_hash}\n\n")
+
+# Store all OT related variables
+get_cmake_property(_variableNames VARIABLES)
+list (SORT _variableNames)
+foreach (_variableName ${_variableNames})
+    if("${_variableName}" MATCHES "^CONFIG_OPENTHREAD_.*|^OT_.*") 
+	    list(APPEND OT_SETTINGS "${_variableName}=${${_variableName}}\n")
+    endif()
+endforeach()
+
+FILE(WRITE "${NRFXLIB_DIR}/openthread/openthread_lib_configuration.txt" ${OT_SETTINGS})
+
+set(OPENTHREAD_LIBS
```

Comment:
I think this could be replaced with a proper GLOB expression. For some reason it was failing for me so I left it as is for now. Having it replaced would probably reduce potential maintenance of this project in case the lib structure changes again...